pkg/reconciler/delivery/delivery.go

```diff
@@ -157,15 +165,15 @@ func (c *Reconciler) fetchPolicyState(cfg *v1.Configuration) (*v1alpha1.PolicySt
 }
 
 // applyChanges applies the newly create Route and PolicyState objects and wraps up the reconciliation
-func (c *Reconciler) applyChanges(ctx context.Context, cfg *v1.Configuration, route *v1.Route, ps *v1alpha1.PolicyState, revisionMap map[string]*v1.Revision) error {
+func (c *Reconciler) applyChanges(ctx context.Context, cfg *v1.Configuration, route *v1.Route, ps *v1alpha1.PolicyState, revisionMap map[string]*v1.Revision, p *Policy) error {
```

Comment:
Why does this take a policy and fetch a policy?

Reply:
`applyChanges` doesn't fetch a policy. Just in case you are talking about the `fetchPolicy` call on line 209, it's in `updateRoute` not in `applyChanges`. I suspect that the way github displays/omits lines in a file is a little confusing :)

Reply:
Oh whoops. you're right it's github squashing the file.